Firestore/core/src/firebase/firestore/core/transaction_runner.mm

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <utility>
+
+#include "Firestore/core/src/firebase/firestore/core/transaction_runner.h"
+#include "Firestore/core/src/firebase/firestore/remote/exponential_backoff.h"
+
+#include "absl/algorithm/container.h"
+
+using firebase::firestore::core::TransactionResultCallback;
```

Comment:
You don't need using declarations for members of the `core` namespace since this file is already defining things in that namespace.

Also, move these inside the other namespace definitions and shorten by dropping the leading `firebase::firestore::`.